### ISSUE TITLE
Get Started on the project.

### ISSUE THREAD

**Issue by arjunjauhari** [arjunjauhari/quora-ques#1]
1) Download dataset.
2) Data exploration
3) Setup repository structure

**Comment by ameynaik** [comment]
Timelines:  (for next two weeks)
**5/13 to 5/30**
	1. Data exploration - quora question.
	2. Download dataset.
	3. Data exploration
		a. Check some corner cases
		b. Max length of questions
		c. Dimensions of data. 
	4. Start with logistic regression. 
		a. Then naïve Bayes
		b. Random trees (a couple of algorithms)
			i. Decision trees 
	5. Define a target and metric for the problem. 
	6. Directory 
		a. Data
			i. Training + dev + test (training kaggle)
			ii. Test (from kaggle)
		b. Model
		c. Notebooks

**Comment by naik-amey** [comment]
![image](https://user-images.githubusercontent.com/8001516/39973063-3b014210-56e0-11e8-9705-863ff420a0b7.png)

**Comment by naik-amey** [comment]
https://www.kaggle.com/anokas/data-analysis-xgboost-starter-0-35460-lb/code 

Summary:
Total number of question pairs for training: 404290
Duplicate pairs: 36.92%
Total number of questions in the training data: 537933
Number of questions that appear multiple times: 111780
Total number of question pairs for testing: 2345796

It is also worth pointing out that the actual number of test rows are likely to be much lower than 2.3 million. According to the data page, most of the rows in the test set are using auto-generated questions to pad out the dataset, and deter any hand-labeling. This means that the true number of rows that are scored could be very low.

For my FYI: 

1. from nltk.corpus import stopwords
    stops = set(stopwords.words("english"))
    https://www.geeksforgeeks.org/removing-stop-words-nltk-python/
